pkg/client/client.go

```diff
@@ -320,13 +320,18 @@ type ProviderUpdateSummary struct {
 func (c *Client) CheckForProviderUpdates(ctx context.Context) ([]ProviderUpdateSummary, error) {
 	var summary []ProviderUpdateSummary
 	for _, p := range c.Providers {
+		// if version is latest it means there is no update as DownloadProvider will download the latest version automatically
+		if strings.Compare(p.Version, "latest") == 0 {
+			c.Logger.Debug("version is latest", "provider", p.Name, "version", p.Version)
```

Comment:
maybe Info level?

Reply:
That is not useful to see at the info level. I would even drop the debug given this all is executed after DownloadProviders, which should complain if it fails to update the provider to the latest version